### ISSUE TITLE
Not finding classes from my project

### ISSUE THREAD

**Issue by designermonkey** [HvyIndustries/crane#376]
If I hold down the command key and click on a class name (like in `new MyClassName`) It gives me information about all possibilities, but not from my current project.

I am on Mac, with the latest VSCode and latest extension version.

**Comment by nevadascout** [comment]
Are you using a multi-folder workspace?

**Comment by designermonkey** [comment]
Nah, just a single folder. I'm sure I've seen it working before, but I can't be definitely sure :(
